packages/forma-36-react-components/src/components/DateTime/index.ts

```diff
@@ -0,0 +1,8 @@
+import { DateTime } from './DateTime/DateTime';
+import { RelativeDate } from './RelativeDate/RelativeDate';
+import * as datetime from './dateUtils';
+export {
+  datetime,
```

Comment:
Not sure the best way to group all of these utility functions, so feedback is welcome

Reply:
This looks good, in my opinion 🤝